src/sid/contacts.py

```diff
@@ -319,8 +333,8 @@ def _calculate_infections_by_recurrent_contacts(
 
     for i in range(n_individuals):
         for cm in range(n_recurrent_contact_models):
-            # We only check if i infects someone else from his group. Whether
-            # he is infected by some j is only checked, when the main loop arrives
+            # We only check if i infects someone else from her/his group. Whether
+            # she/he is infected by some j is only checked, when the main loop arrives
```

Comment:
Why stop halfway? :)
```suggestion
            # We only check if i infects someone else from their group. Whether
            # they is infected by some j is only checked, when the main loop arrives
```